### PR TITLE
Rabbitmq helm

### PR DESCRIPTION
**Pull request by akshat5302**

### CODE REVIEW

File: charts/plane-ce/values.yaml [hunk 1]
```diff
@@ -41,6 +41,17 @@ postgres:
   volumeSize: 5Gi
   pullPolicy: IfNotPresent
 
+rabbitmq:
+  local_setup: true
+  image: rabbitmq:3-management
```
Comment (mguptahub) [inline charts/plane-ce/values.yaml:46]: change the image to `rabbitmq:3.13.6-management-alpine`

File: charts/plane-ce/templates/workloads/rabbitmq.stateful.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: {{ .Release.Namespace }}
+  name: {{ .Release.Name }}-rabbitmq
+  labels:
+    app.name: {{ .Release.Namespace }}-{{ .Release.Name }}-rabbitmq
+spec:
+  clusterIP: None
```
Comment (mguptahub) [inline charts/plane-ce/templates/workloads/rabbitmq.stateful.yaml:11]: something missing. 
  ```
  {{- if not .Values.rabbitmq.assign_cluster_ip }}
  clusterIP: None
  {{- end }}
```

File: charts/plane-ce/values.yaml [hunk 3]
```diff
@@ -41,6 +41,17 @@ postgres:
...
+  servicePort: 5672
+  managementPort: 15672
+  storageClass: longhorn
+  volumeSize: 500Mi
+  default_user: plane
+  default_password: plane
+  external_rabbitmq_url: ''
+
```
Comment (mguptahub) [inline charts/plane-ce/values.yaml:54]: add `assign_cluster_ip: false`

**Review by mguptahub — COMMENTED** [review]
Please pull latest `develop` branch and then recheck the merge request

**Review by mguptahub — CHANGES_REQUESTED** [review]
Please re-pull the `develop` branch in your branch and check the PR file changes

File: charts/plane-ce/values.yaml (hunk 1 above)
Comment (akshat5302) [reply charts/plane-ce/values.yaml]: changed to `rabbitmq:3.13.6-management-alpine`

File: charts/plane-ce/templates/workloads/rabbitmq.stateful.yaml (hunk 2 above)
Comment (akshat5302) [reply charts/plane-ce/templates/workloads/rabbitmq.stateful.yaml:12]: added if block

**Review by mguptahub — APPROVED** [review]

**Pull request merged**